docs/index.md

```diff
@@ -24,19 +24,17 @@ The BitBox Base is an ongoing project of [Shift Cryptosecurity](https://shiftcry
    1. [Configuration](os/configuration.md)
    1. [Security considerations](os/security.md)
    1. [Helper scripts](os/helper-scripts.md)
-1. Base Middleware
-   1. API
-   1. Noise encryption
-   1. HSM integration
+1. Go applications
```

Comment:
You might want to pick 91b1e7644f0d42ad09fa36fd017471217824d0b6 into this branch to avoid a conflict when rebasing after #69 is merged.